### ISSUE TITLE
UX - thanks page looks too similar to home page

### ISSUE THREAD

**Issue by jsms90** [pbywater/week6-zapo#52]
At first, I didn't realise that I had been taken to a "thank you" page, because the layout is almost identical:
+ same background pic
+ ZAPO Recipes is the most prominent thing
+ number & colour of buttons
+ spacing

This makes it seem like the user has just been taken back to the homepage, and could leave them unsure as to whether their recipe has submitted successfully.

If I have made a payment on a website, I would expect to see a thank you page. But if I've submitted a recipe, I think I would rather go directly to viewing the recipe that I just submitted? This could be with an added thank you at the top, to make it clear that something has happened.

**Comment by oliverjam** [comment]
You're right, we didn't get any time to focus on UX/CSS so we just duplicated that section for the thanks page
